consultation_analyser/consultations/models.py

```diff
@@ -42,7 +42,7 @@ class Consultation(UUIDPrimaryKeyModel, TimeStampedModel):
     users = models.ManyToManyField(User)
 
     def has_themes(self):
```

Comment:
Should this be `has_been_processed` or sth now? i.e. checking for presence of a run, not themes?

Reply:
Good idea - has been changed to `has_been_processed`.